Packs/PagerDuty/Integrations/PagerDuty/PagerDuty.py

```diff
@@ -486,6 +492,35 @@ def extract_users_notification_role(user_notification_role):
     }
 
 
+def extract_responder_request(responder_request_response):
+    """Extract the users that were requested to respond"""
+    outputs = []
+    responder_request = responder_request_response.get("responder_request")
+    for request in responder_request.get("responder_request_targets", []):
+        output = {}
+        output["Type"] = request.get("type")
+        output["ID"] = request.get("id")
+        responder_user = request.get("incident_responders", {}).get("user", {})
+        output["ResponderID"] = responder_user.get("id")
+        output["ResponderType"] = responder_user.get("type")
+        output["ResponderName"] = responder_user.get("summary")
+        output["Message"] = responder_request.get("message")
+        output["IncidentID"] = responder_request.get("incident", {}).get("id")
+        output["IncidentSummary"] = responder_request.get("incident", {}).get("summary")
+        outputs.append(output)
+    return {
+        'Type': entryTypes['note'],
+        'Contents': outputs,
+        'ContentsFormat': formats['json'],
+        # 'ReadableContentsFormat': formats['markdown'],
+        # 'HumanReadable': tableToMarkdown(CONTACT_METHODS, outputs, CONTACT_METHODS_HEADERS),
```

Comment:
why both of these lines are commented?

Reply:
I don't know I just took over the code of my teammate .

Reply:
Changed in https://github.com/demisto/content/pull/17745/commits/072e7428c67b5d89c3c1bb6f178b2ac1116c04f4